refract-spec.md

```diff
@@ -12,89 +12,286 @@ This document conforms to RFC 2119, which says:
 
 [MSON](https://github.com/apiaryio/mson) is used throughout this document to describe and define data structures.
 
-## Data Structure
+## Primitive Types
 
-Refract provides a single structure for data, which will be referred to throughout this document as an element. All elements found in a Refract document SHOULD conform to the element structure as defined here.
+Primitive Types are types used by Refract as the basis for all values and structure. These types use both JSON primitive values along with Refract Elements.
 
-### Element
+Refract provides Primitive Types and Primitive Elements for the purpose of expanding JSON primitive values and adding meta data and attributes. The allows Refract to be infinitely recursive so meta data and attributes can always be added to any value.
 
-The Refract Element contains three properties, `element`, `attributes`, and `content`, as defined below. This Element can be used recursively throughout the document, even as a value for each of its own properties.
+### Value Types
 
-#### Properties
+Value types are primitive types that Refract uses that do not define structure. Value types consist of:
 
-- `element` (enum, required)
+* Null Type
+* String Type
+* Number Type
+* Boolean Type
 
-    The `element` property defines the type of element. It MAY be a string that references an element type, or it MAY be an `element` itself that provides additional data about the element type.
+#### Null Type (enum)
 
-    - Members
-        - (string)
+A Null Type in Refract can be either the `null` value or a Null Element.
+
+##### Members
```

Comment:
Why this vs. just using MSON list right after that defines the enum members? Is this to be more human friendly? It is more verbose than it needs to be from an MSON standpoint.